spec/factories/stripe_records.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :stripe_record do
+    transient do
+      create_child { false }
```

Comment:
Either better name `create_child_charge` or more flexibility (default `nil`, pass in a trait like `:charge`, `:refund`, etc. on-demand)

Reply:
Fixed